src/overlays/actors/ovl_Obj_Oshihiki/z_obj_oshihiki.c

```diff
@@ -84,7 +84,7 @@ s32 ObjOshihiki_StrongEnough(ObjOshihiki* this, GlobalContext* globalCtx) {
         return true;
     }
 
-    if (func_80124020() >= 2) {
+    if (Player_GetStrength() >= 2) {
```

Comment:
Was there going to be an enum for this?

Reply:
Ohh right, I forgot to make it...
I'll add it